lib/granite/represents/attribute.rb

```diff
@@ -1,6 +1,10 @@
 module Granite
   module Represents
     class Attribute < ActiveData::Model::Attributes::Attribute
+      TYPES = {
+        ActiveRecord::Enum::EnumType => String
```

Comment:
We should either add AR into gemspec or make it conditional

Reply:
Also, shouldn't it go to `active_data`? It already has enums https://github.com/pyromaniac/active_data#enums, but I guess hash syntax is not supported.

Reply:
`enum` still uses `String` for the attribute type, it just adds `options[:enum]`. We could override `enumerizer` in `Granite::Represents::Attribute`, but `ActiveRecord::Enum::EnumType` doesn't provide public API for getting valid values.